src/raja/RAJAStream.cpp

```diff
@@ -9,67 +9,30 @@
 #include <stdexcept>
 #include "RAJAStream.hpp"
 
-using RAJA::forall;
-
-#ifndef ALIGNMENT
-#define ALIGNMENT (2*1024*1024) // 2MB
-#endif
-
 template <class T>
 RAJAStream<T>::RAJAStream(const int ARRAY_SIZE, const int device_index)
     : array_size(ARRAY_SIZE), range(0, ARRAY_SIZE)
 {
-
-#ifdef RAJA_USE_CHAI
-  d_a = new chai::ManagedArray<T>(array_size);
-  d_b = new chai::ManagedArray<T>(array_size);
-  d_c = new chai::ManagedArray<T>(array_size);
-  std::cout << "Using CHAI. All allocations complete." << std::endl;
-#elif RAJA_TARGET_CPU
-  d_a = (T*)aligned_alloc(ALIGNMENT, sizeof(T)*array_size);
-  d_b = (T*)aligned_alloc(ALIGNMENT, sizeof(T)*array_size);
-  d_c = (T*)aligned_alloc(ALIGNMENT, sizeof(T)*array_size);
-#else
-  cudaMallocManaged((void**)&d_a, sizeof(T)*ARRAY_SIZE, cudaMemAttachGlobal);
-  cudaMallocManaged((void**)&d_b, sizeof(T)*ARRAY_SIZE, cudaMemAttachGlobal);
-  cudaMallocManaged((void**)&d_c, sizeof(T)*ARRAY_SIZE, cudaMemAttachGlobal);
-  cudaDeviceSynchronize();
-  std::cout << "NOT using CHAI. Allocations complete." << std::endl;
-#endif
+    d_a = static_cast<T*>(alloc.allocate(sizeof(T) * array_size));
+    d_b = static_cast<T*>(alloc.allocate(sizeof(T) * array_size));
+    d_c = static_cast<T*>(alloc.allocate(sizeof(T) * array_size));
 }
 
 template <class T>
 RAJAStream<T>::~RAJAStream()
 {
-#ifdef RAJA_USE_CHAI
-  delete d_a;
-  delete d_b;
-  delete d_c;
-#elif RAJA_TARGET_CPU
-  free(d_a);
-  free(d_b);
-  free(d_c);
-#else
-  cudaFree(d_a);
-  cudaFree(d_b);
-  cudaFree(d_c);
-#endif
+    rm.getAllocator(d_a).deallocate(d_a);
+    rm.getAllocator(d_b).deallocate(d_b);
+    rm.getAllocator(d_c).deallocate(d_c);
```

Comment:
Indentation trouble. Looks like this file uses 2 spaces to indent, these lines are four spaces.